learningequality/studio
# Release PR for Sprint ending 2021-03-15

## Release PR for Sprint ending March 16

### Fixes for:

#2989, #2990 and #1404 

For Fix 1

Using the user@c.com created by the devsetup script, I invite a@a.com to be viewer of "empty channel".
After logging in as a@a.com, I decline the invitation.
After logging back in as user@c.com, I see that a@a.com's invite is gone (previously, it would show up upon logging in and going to the sharing page, but would disappear after a refresh)
For Fix 2

(Use two different browsers, e.g. firefox and chrome for each account)

As c, invite a to view a channel (do not leave the Sharing page)
As a, accept the invitation
As c, refresh the page to see that a has accepted the invitation (at this point, the invitation might still be in vuex in a stale unaccepted state).
As c, remove a's view access
a should disappear from the table

#2888 (part 1) 

1. Login to Studio
2. Click a Channel
3. Click the pencil icon to load the channel's edit details page
4. Make some amendments of the channel's content defaults
5. Click Save Changes button
6. Make sure to close the channel's edit details page before /sync gets called
7. Wait for /sync call to happen (otherwise you will run into (2) which has not been fixed yet)
8. Click the pencil icon to load the channel's edit details page
9. See the changes made are visible in the channel edit details page (note: this PR does _not_ include fixes for separate but related sync issues on the main Channel page. Manual QA should only ensure the _form_ changes are working correctly. )lease see issue and related PR for further details if needed.)

#2998 

Testing with the Admin Table users tab

1. Ensure you db has a number of users with editing permissions on various channels
2. Sign in as an admin
3. On the /administration page, go the the users tab and search
4. The correct count and users should return, and the pagination should be correct
